.helm/templates/crd-microsoft-synapse.yaml

```
@@ -270,6 +270,22 @@ spec:
                   properties:
                     name:
                       type: string
+                fieldSelectionRule:
+                  type: object
+                  properties:
+                    type:
```

```suggestion
                    ruleType:
```
to avoid name clashes